src/@types/MetaData.d.ts

```diff
@@ -39,6 +40,7 @@ export interface MetadataPublishForm {
   files: string | File[]
   author: string
   timeout: string
+  price?: number
```

Comment:
don't think we need `price` on here, since in our publish form we do not capture the price so should be removed from here